README.md

```diff
@@ -1,39 +1,14 @@
-Pull Request Template
-=====================
+attn: Pull Request Template
```

Comment:
Company name should be capitalized when written out as per [our brand guidelines](https://sites.google.com/a/attn.com/documentation/home/organization/brand-guidelines). Alternately, we could throw the logo in [like this](https://github.com/attn/automation/blob/master/README.md).